js/components/Toolbox.js

```diff
@@ -0,0 +1,28 @@
+const React = require("react");
```

Comment:
`Toolbox` might be too confusing. It implies a toolbar where you can select multiple tools. I'd prefer a different name, but I don't know what. `TabList`?